meta-resin-common/recipes-support/resinhup/resinhup/run-resinhup.sh

```diff
@@ -359,7 +359,7 @@ if version_gt $HOSTOS_VERSION "1.1.5" || [ "$HOSTOS_VERSION" == "1.1.5" ]; then
         log "Running engine migrator 1.10... please wait..."
         DOCKER_MIGRATOR="registry.resinstaging.io/resinhup/$arch-v1.10-migrator"
         $DOCKER pull $DOCKER_MIGRATOR
-        $DOCKER run --rm -v /var/lib/rce:/var/lib/docker $DOCKER_MIGRATOR -s btrfs
+        $DOCKER run --rm -v /var/lib/rce:/var/lib/docker $DOCKER_MIGRATOR -s aufs
```

Comment:
Yes this is correct. Or a re-download from the registry. Either case will be handled by resinhup so we shouldn't worry about it here.